### PR TITLE
Docs/finish readme

### PR DESCRIPTION
**Pull request by greenbech**
Add GIF of the finished product.

Add IMC document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,25 @@
...
 
+## Installation
+
+If you just want to run the project, go to the [release tab](https://github.com/eirikdahlen/TDT4290_Aqueous/releases) in this GitHub repository and it should be plug and play.
+
+Every release of the product is available here as an installable `.exe` file (for Windows), along with the features included in each release.
+
+Simply download the `AqueousInstaller.exe` file, run as Administrator and run the program as any other desktop application.
+
+If you are on any other operation system you will have to build the project your self. It is eary (se below).
```
Comment (eirikdahlen) [inline README.md]: easy?

Comment (Trebua) [inline README.md]: eary?

Comment (Trebua) [reply README.md]: se?

Comment (eirikdahlen) [reply README.md]: Also it is yourself* in one word

Comment (Trebua) [reply README.md]: Typos:
* your self = yourself
* eary = easy?
* se = see
Maybe rephrase last part to "See section .."

File: README.md [hunk 2]
```diff
@@ -89,3 +95,18 @@ By doing the following steps, two computers can communicate over TCP by using th
...
 
 Follow the conventions specified in [CONTRIBUTION](./CONTRIBUTING.md).
+
+## The IMC message protocol
+
+We use the IMC message protocol to communicate with the ROV. Documentation for our custom messages, and much more, can be found [in this document](./assets/IMC-protocol/IMC-Proposition.pdf).
+
+## Running the project without the simulator
+
+The simulator we used in the GIF above were provided by SINTEF Ocean and it not publically avaliable. It it however possible to try this GUI with a mockup we provided.
```
Comment (eirikdahlen) [inline README.md]: It is* however

File: README.md (hunk 1 above)
Comment (Trebua) [reply README.md]: see*
Also rephrase last part to "See section .."

File: README.md [hunk 3]
```diff
@@ -89,3 +95,18 @@ By doing the following steps, two computers can communicate over TCP by using th
...
+
+1. Go go `View -> Show IMC-ROV Mockup -> Click "Start Server"`
+   _This starts the ROV server and it now waits for a connection. It should now display "Server is running..."_
+2. Make sure that the message protocol is set to `IMC`. This can be done by going to `File -> Settings -> Change protocol to IMC (not OLD)`.
+3. Click `Rov -> Connect to TCP`.
+
+You should now see vales in the bottum of the mockup window. If you activate the video window you should also be able to send values by using the keyboard mappings (which can be shown by `Control -> Show keyboard controls`).
```
Comment (Trebua) [inline README.md]: values*
bottom*

**Comment by greenbech** [comment]
> See comments on typing errors!
> Other than that, very nice!
> 
> Maybe also Contributing should be at the bottom?

Agree that this is better. 

I also fixed a lot of typos and commas with a grammar checker
